### PR TITLE
adjust for openfunction release v0.8.0-rc.0

### PR DESCRIPTION
**Pull request by wrongerror**
Signed-off-by: wrongerror <wangyifei@kubesphere.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # openfunction
 
-![Version: 0.2.0](https://img.shields.io/badge/Version-0.2.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.7.0](https://img.shields.io/badge/AppVersion-0.7.0-informational?style=flat-square)
+![Version: 0.3.0-rc.0](https://img.shields.io/badge/Version-0.3.0--rc.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.0-rc.0](https://img.shields.io/badge/AppVersion-0.8.0--rc.0-informational?style=flat-square)
```
Comment (benjaminhuo) [inline README.md:3]:
```suggestion
![Version: 0.3.0-rc.0](https://img.shields.io/badge/Version-0.3.0--rc.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.0-rc.0](https://img.shields.io/badge/AppVersion-0.8.0-rc.0-informational?style=flat-square)
```

Comment (benjaminhuo) [inline README.md:3]: `Version-0.3.0--rc.0`
`AppVersion-0.8.0--rc.0`

there is an extra `-` , is that ok?

Comment (wrongerror) [reply README.md:3]: This extra `-` is needed.

Comment (wrongerror) [reply README.md:3]: https://img.shields.io/badge/AppVersion-0.8.0--rc.0-informational?style=flat-square

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**